test/gfx/test.sh

```diff
@@ -37,6 +37,26 @@ failTest () {
 	echo "${bold}${red}Test ${cmdline} failed!${1:+ (RC=$1)}${rescolors}${resbold}"
 }
 
+checkOutput () {
+	out_rc=0
+	if [[ -e "$1.out.bin" ]]; then
+		cmp "$1.out.bin" result.2bpp
+		(( out_rc = out_rc || $? ))
+	fi
+	if [[ -e "$1.out.tilemap" ]]; then
+		cmp "$1.out.tilemap" result.tilemap
+		(( out_rc = out_rc || $? ))
+	fi
+	if [[ -e "$1.out.attrmap" ]]; then
+		cmp "$1.out.attrmap" result.attrmap
+		(( out_rc = out_rc || $? ))
+	fi
+	if [[ -e "$1.out.pal" ]]; then
+		cmp "$1.out.pal" result.pal
+		(( out_rc = out_rc || $? ))
+	fi
```

Comment:
```suggestion
	for ext in 2bpp 1bpp tilemap attrmap pal palmap; do
		if [[ -e "$1.out.$ext" ]]; then
			cmp "$1.out.$ext" result.$ext
			(( out_rc = out_rc || $? ))
		fi
	done
```

... and a little `rename bin 2bpp test/gfx/*.out.bin`.

Reply:
The .2bpp extension would get picked up as inputs for the reverse-mode tests.

Reply:
I added a check in a further suggestion to guard against that (skipping files that match `*.out.[12]bpp`.